meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
If it's not the default I don't think anyone would use it. How verbose is it though? I just fear that putting this up on a big screen during a presentation at ELC, it's gonna look bad if the screen is flooded with log messages.